Source/Core/Core/HW/DSPHLE/UCodes/AXWii.cpp

```diff
@@ -29,6 +29,7 @@ AXWiiUCode::AXWiiUCode(DSPHLE* dsphle, u32 crc)
     volume = 0x8000;
 
   m_old_axwii = (crc == 0xfa450138) || (crc == 0x7699af32);
+  m_new_filter = crc == 0x347112ba || crc == 0x4cc52064;
```

Comment:
It looks a bit silly to have parens for one but not the other. (I don't have a strong preference between them though).